ibis/backends/oracle/tests/test_client.py

```diff
@@ -67,14 +67,11 @@ def stats_one_way_anova(x, y, value: str) -> int:
     tm.assert_frame_equal(result, expected, check_dtype=False)
 
 
-def test_list_tables_schema_warning_refactor(con):
+def test_list_tables(con):
     assert con.list_tables()
 
     with pytest.raises(exc.IbisInputError):
-        con.list_tables(database="not none", schema="not none")
-
-    with pytest.warns(FutureWarning):
-        assert con.list_tables(schema="SYS", like="EXU8OPT") == ["EXU8OPT"]
+        con.list_tables(database="not none")
```

Comment:
This seems to be broken, but I can't run the Oracle test suite locally, I'll need some help debugging this one.